src/classes/UTIL_Describe_TEST.cls

```diff
@@ -189,6 +223,39 @@ public with sharing class UTIL_Describe_TEST {
         System.assert(recTypeInfo == null);
     }
 
+    /***********************************************************************************************
+    * @description Test to verify that getRecordTypeInfoByName() does not return null when
+    * there are no Account record types assigned and an invalid Record Type is used.
+    ***********************************************************************************************/    
+    @isTest
+    private static void getRecordTypeInfoByNameStandardUserPositive() {
+        User stdUser = UTIL_Describe_TEST.getStandardUser();
+
+        System.runAs(stdUser){
+            Test.startTest();
+            Schema.RecordTypeInfo recTypeInfo = Util_Describe.getRecordTypeInfoByName('Account', 'Academic Program');
+            Test.stopTest();
+
+            System.assertEquals(true, recTypeInfo != null, 'Should retrieve record type info for valid record type even if not assigned.');
+        }
+    }
+ 
+    /***********************************************************************************************
+    * @description Test to verify that getRecordTypeInfoByName() returns null when
+    * there are no Account record types assigned and an invalid Record Type is used.
+    ***********************************************************************************************/
+    @isTest
+    private static void getRecordTypeInfoByNameStandardUserNegative() {
```

Comment:
This is one of the scenarios where we have already covered this use case. The standard user additional scenarios can be limited to valid with assigned and unassigned. Since it's here we don't necessarily need to remove it. The extra validation will not hurt us.